### ISSUE TITLE
1.5.3 fails on vars parsing

### ISSUE THREAD

**Issue by antongolub** [joarwilk/flowgen#68]
Hi there. 

Caught an error on `const` declaration parsing:

Input index.d.ts:
```javascript
declare module '@qiwi/cyclone/lib/es5/error' {
	export const TRANSITION_VIOLATION: string;
	export const INVALID_UNLOCK_KEY: string;
	export const LOCK_VIOLATION: string;
	export const UNREACHABLE_STATE: string;
	export class MachineError extends Error {
	}
}
```
Exception:
```javascript
TypeError: factory.createVariableNode is not a function
    at collectNode (/Users/antongolub/projects/cyclone/node_modules/flowgen/lib/parse.js:119:57)
    at /Users/antongolub/projects/cyclone/node_modules/flowgen/lib/parse.js:152:14
    at Array.forEach (<anonymous>)
    at traverseNode (/Users/antongolub/projects/cyclone/node_modules/flowgen/lib/parse.js:151:21)
    at collectNode (/Users/antongolub/projects/cyclone/node_modules/flowgen/lib/parse.js:98:9)

```
After refactoring the previous `parse.js` implementation (which refers to factory.createVariableNode) is still present in package. Once `parse.js` is deleted, everything works as it should.

<img width="324" alt="screenshot 2019-02-05 at 11 42 50" src="https://user-images.githubusercontent.com/5288046/52262102-85bc5680-293c-11e9-9396-22a5bcf21e14.png">

Could anybody repack / publish a new version?

**Comment by goodmind** [comment]
@orta

**Comment by orta** [comment]
Ah, I see, I’ll edit the build script to rm the old dir first

**Comment by orta** [comment]
#69  beat me to it, shipped as 1.5.5
